associations/belongs_to_association_test.go

```diff
@@ -21,21 +21,20 @@ func (f fooBelongsTo) TableName() string {
 }
 
 type barBelongsTo struct {
-	FooBelongsToID uuid.UUID    `db:"foo_id"`
-	Foo            fooBelongsTo `belongs_to:"foo"`
-	NestedBar      nestedBar    `has_one:"nestedBar"`
+	FooID uuid.UUID    `db:"foo_id"`
+	Foo   fooBelongsTo `belongs_to:"foo"`
 }
 
 type nestedBar struct {
```

Comment:
@stangah not sure about the intention  for `NestedBar` struct here and it seems that it does not make any sense to defining it on this test case. Would you mind to remove it, since it is not used anymore?

Reply:
Sure, will do